Source/Krypton Toolkit/Main/Krypton.Toolkit.Suite.Extended.Messagebox/Classes/InternalKryptonMessageBoxExtended.cs

```diff
@@ -256,7 +256,7 @@ private void InitialiseComponent()
         private MessageButton _copyButton;
         private KryptonCheckBox _optionalCheckBox;
         private KryptonBorderEdge _borderEdge;
-        private HelpInformation _helpInformation; // TODO: What is this used for ?
+        private HelpInformation _helpInformation;
```

Comment:
Why was the TODO removed ? It has not been implemented / described !

Reply:
@Smurf-IV I found that the `HelpInformation` is a nested class inside of the `KryptonMessageBox` class which deals with help files. It seems unnecessary, so should I rip it out?

Reply:
No, Not rip it out.
 The TODO comment needs to be left, because the Winfrom-MessageBox takes an optional HelpInformation item. It's just that the KMB never implemented it, so the TODO needs to be there as a reminder.

Reply:
Reinstated this